sdks/python/apache_beam/runners/worker/operation_specs.py

```diff
@@ -332,16 +334,16 @@ def get_coder_from_spec(coder_spec):
     return coders.IntervalWindowCoder()
   elif coder_spec['@type'] == 'kind:global_window':
     assert ('component_encodings' not in coder_spec
-            or not coder_spec['component_encodings'])
-    return coders.GlobalWindowCoder()
+            or len(coder_spec['component_encodings'] == 0))
```

Comment:
I think `)` is misplaced and it should read `len(coder_spec['component_encodings']) == 0)`